tests/unit/test_utils.py

```
@@ -477,6 +475,47 @@ def test_invalid_samplesheet_logged_and_continues(self, mock_read):
 
         rmtree(sequencer_output_dir)
 
+    def test_runs_with_not_all_samples_matching_sample_pattern_skipped(self):
```

Ruff (0.7.0)

Missing return type annotation for public function `test_runs_with_not_all_samples_matching_sample_pattern_skipped`

Add return type annotation: `None`

(ANN201)